aptos-move/aptos-vm/Cargo.toml

```diff
@@ -17,6 +17,7 @@ mirai-annotations = "1.10.1"
 once_cell = "1.7.2"
 rayon = "1.5.0"
 tracing = "0.1.16"
+envmnt = "0.9.1"
```

Comment:
we don't need this, just use std::env?

Reply:
It only returns strings, so seemed like more code duplication. But I thought maybe we can add one more static function to parallel executor and call it twice, and over there I can use env and do parsing and everything.